### DISCUSSION TITLE
Routing on off not working?

### DISCUSSION THREAD

**Discussion by ChotaFallen** [oddbear/GoXLR-TouchPortal-Plugin#29] [General]
Good morning, 
I love your plugin is a great piece of software.
I have a question related to the ne plugin.
I have a button made to turn of eg. routing music to the stream. But it is not working.
If I use toggle instead turn off, I can turn off the routing correctly.

I want that turn off routing, when I change to my gaming scene and it turns off music to the stream.

Thank you

**Comment by oddbear** [comment]
Interesting.

What version of the plugin are you using. There was a breaking change in the App at some point.

v0.0.7 or higher should be working if it's that particular bug.

v0.17.0-pre is the newest that has statuses in it, but does have a lot of breaking changes, so everything in Touch Portal needs to be set up again from scratch.

**Comment by ChotaFallen** [comment]
I reinstalled all, the v0.17.0-pre, build it in Touch Portal from scratch, done it 2 times.
And yeah it is working now 😍😘😘😘
Another reason not to change my GoXLR mini, because with this plugin I have real great submix in addition with another obs plugin.
Thank you for your work.

**Reply by oddbear** [reply]
That is great, and thanks. 😊

**Reply by ChotaFallen** [reply]
Hm strange. It didn´t work, I was wrong. The working button was only toggle. I hadn´t changed it.
Even with extra button only for turn on and turn off won´t work.
You can look via teamviewer if you like maybe I do something wrong?

**Reply by oddbear** [reply]
We can do that, but first I wanna test it out a little for myself to see if I can reproduce it.
I have not done anything on this plugin for a while, since I have been busy working on a Revelator io24 plugin, so I will to connect everything up again.

**Reply by ChotaFallen** [reply]
Nice a io23 plug in. I orderd the mixer yesterday can test it too 🤗

**Reply by oddbear** [reply]
I have reproduced the issue at my end, and might have solved it, try out the new release: https://github.com/oddbear/GoXLR-TouchPortal-Plugin/releases/tag/v0.17.1-pre

**Reply by ChotaFallen** [reply]
At first look. It works properly.
I will test in my stream today.
